test/com/google/enterprise/adaptor/database/JdbcFixture.java

```diff
@@ -47,13 +78,34 @@
    * The default escape character is disabled, to match the DQL
    * behavior (and standard SQL).
    */
+  private static void loadDefaultProperies(Properties properties) {
+    properties.setProperty("test.database", "h2");
+    properties.setProperty("test.driver", "org.h2.Driver");
+    properties.setProperty("test.url",
+        "jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DEFAULT_ESCAPE=");
+    properties.setProperty("test.user", "sa");
+    properties.setProperty("test.password", "");
+  }
+
+  /**
+   * Gets a JDBC connection to the database.
+   */
   public static Connection getConnection() {
     try {
-      JdbcDataSource ds = new JdbcDataSource();
-      ds.setURL(URL);
-      ds.setUser(USER);
-      ds.setPassword(PASSWORD);
-      return ds.getConnection();
+      if (database.equalsIgnoreCase("h2")) {
+        JdbcDataSource ds = new JdbcDataSource();
+        ds.setURL(url);
+        ds.setUser(user);
+        ds.setPassword(password);
+        return ds.getConnection();
+      } else {
+        try {
+          Class.forName(driver_class);
+          return DriverManager.getConnection(url, user, password);
+        } catch (ClassNotFoundException e) {
```

Comment:
Not worth having a separate try statement, just roll ClassNotFoundException into the existing catch clause.